wasm/module.go

```diff
@@ -130,61 +136,3 @@ func DecodeModule(binary []byte) (*Module, error) {
 	}
 	return ret, nil
 }
-
-func (m *Module) DecodeCustomNameSection() (map[uint32]string, error) {
```

Comment:
ps I ported this logic, but it is left alone. I'll revisit it in another PR so that we can read all the standard subsections